.github/workflows/quality.yml

```diff
@@ -74,7 +74,7 @@ jobs:
           edit-mode: replace
       - name: Delete comment if needed
         if: always() && steps.fc.outputs.comment-id != '' && steps.ktlint-results.outputs.add_comment == 'false'
-        uses: actions/github-script@v3
+        uses: actions/github-script@v5.1.0
```

Comment:
This should not be updated.

Reply:
I opened a [PR](https://github.com/vector-im/element-android/pull/4989) with the appropriate fix